mlkem/kem.c

```diff
@@ -116,6 +124,7 @@ __contract__(
   requires(memory_no_alias(sk, MLKEM_INDCCA_SECRETKEYBYTES)));
 
 #if defined(MLK_KEYGEN_PCT)
+/* Specification: Implements 'Pairwise Consistency Test' [FIPS 203 IG]. */
```

Comment:
Actually you can also reference FIPS 203 Section 7.1. (Pair-wise consistency)
Also you there is no such thing as an FIPS 203 IG - you mean the FIPS 140-3 IG I suppose?

Reply:
You're right about 7.1 and the IG citation.

Sadly, this makes it plain that the current fixed-coins implementation of the PCT is not conformant -- we need to think about how to address this.